src/tf.h

```diff
@@ -80,25 +80,26 @@ enum enum_attr {
     F_FLASH       = 0x0000,   /* zero - not implemented */
     F_DIM         = 0x0000,   /* zero - not implemented */
     F_BOLD        = 0x0004,
-    F_HILITE      = 0x0008,
-    F_NONE        = 0x0010,
-    F_EXCLUSIVE   = 0x0020,
+    F_ITALIC      = 0x0008,
+    F_HILITE      = 0x0010,
+    F_NONE        = 0x0020,
+    F_EXCLUSIVE   = 0x0040,
 
 #if NCOLORS == 256 /* XXX ??? */
 # define FGCOLORSHIFT 8
     F_FGCOLORMASK = 0x0000ff00,   /* 8 bits, interpreted as an integer */
-    F_FGCOLOR     = 0x00000040,   /* flag */
+    F_FGCOLOR     = 0x00000080,   /* flag */
 # define BGCOLORSHIFT 16
     F_BGCOLORMASK = 0x00ff0000,   /* 8 bits, interpreted as an integer */
-    F_BGCOLOR     = 0x00000080,   /* flag */
+    F_BGCOLOR     = 0x00000100,   /* flag */
```

Comment:
There's a regression with this change I happened to notice:

This flag now overlaps F_FGCOLORMASK. That means that if foreground color is set and happens to be an odd number, background color will also be set.